docs/conf.py

```diff
@@ -1,188 +1,69 @@
-# -*- coding: utf-8 -*-
+# Configuration file for the Sphinx documentation builder.
 #
-# Core ML documentation build configuration file
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-import coremltools
-import sys
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
 import os
-import re
+import sys
+sys.path.insert(0, os.path.abspath('.'))
 
-for m in [
-    "converters",
-    "utils",
-]:
-    module_name = "coremltools." + m
-    sys.modules[module_name] = eval(module_name)
 
-sys.path.insert(0, os.path.abspath("."))
+# -- Project information -----------------------------------------------------
 
-# -- General configuration ------------------------------------------------
+project = 'coremltools'
+copyright = '2021, Apple Inc.'
+author = 'tbove'
 
-# If your documentation needs a minimal Sphinx version, state it here.
-#
-# needs_sphinx = '1.0'
+# The full version, including alpha/beta/rc tags
+release = '4.1'
+
+
+# -- General configuration ---------------------------------------------------
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "sphinx.ext.autodoc",
+   "sphinx.ext.autodoc",
```

Comment:
Probably don't want this change.

Reply:
Thanks. I'll make this change for the next PR.